### DISCUSSION TITLE
How to autosave changes?

### DISCUSSION THREAD

**Discussion by tonmoy998** [LazyVim/LazyVim#1022] [Q&A]
I am doing web development and using live-server for it.Please give me suggestions to autosave lazyvim

**Comment by LintaoAmons** [comment]
Can check my autocmd. https://github.com/LintaoAmons/CoolStuffes/blob/5b117d662f1f64e528545d584f5781f4522092c7/lazyvim/.config/nvim/lua/config/autocmds.lua#L5

```
local function save()
  local buf = vim.api.nvim_get_current_buf()

  vim.api.nvim_buf_call(buf, function()
    vim.cmd("silent! write")
  end)
end

vim.api.nvim_create_augroup("AutoSave", {
  clear = true,
})

vim.api.nvim_create_autocmd({ "InsertLeave", "TextChanged" }, {
  callback = function()
    save()
  end,
  pattern = "*",
  group = "AutoSave",
})
```

**Reply by tonmoy998** [reply]
Don't know why not working for me.

**Reply by nicocossiom** [reply]
very nice, thank you! it works very well, this should be an extra enabled in the default Lazyvim config
